Sources/ArgumentParserToolInfo/ToolInfo.swift

```diff
@@ -50,7 +116,11 @@ public struct CommandInfoV0: Codable, Hashable {
   /// Short description of the command's functionality.
   public var abstract: String?
   /// Extended description of the command's functionality.
-  public var discussion: String?
+  ///
+  /// Can include specific abstracts about the argument's possible values (e.g.
+  /// for a custom `CaseIterable` type), or can describe
+  /// a static block of text that extends the description of the argument.
+  public var discussion: Discussion?
```

Comment:
Yeah this will need a version bump and will be source break. Thats not a deal breaker, but can be annoying.

Alternatively we can call this `discussion2` and deprecate the old one.